frontend/src/routes/person/[id].svelte

```diff
@@ -1,5 +1,5 @@
 <script lang="ts">
-    import { removeContentWithMissingImagePath, routeToPage, sortListByPopularity  } from '../../utils'
+    import { removeContentWithMissingImagePath, routeToPage, sortListByPopularity, removeDuplicates, removeDuplicates } from '../../utils'

```

Comment:
Why is `removeDuplicates` imported twice?

Reply:
Uhm, yes.

Reply:
Removed.